pyproject.toml

```diff
@@ -13,7 +13,7 @@ pydantic = ">=1.10.7, <3.0"
 backoff = ">=1.10.0"
 openai = { version = ">=0.27.8", optional = true }
 wrapt = "^1.14"
-langchain = { version = ">=0.0.309", optional = true }
+langchain-core = { version = ">=0.2.0", optional = true }
```

Comment:
Thanks a lot for the contribution!
Do you know how backwards compatible this is? We might have users who explicitly installed langchain version 0.0.310. I assume that the upgrade would not work for them, correct?

Reply:
If yes, is there a way to make this backwards compatible?

Reply:
- Users should install `langfuse` with the `langchain` extra (and this will ensure they have the `langchain-core` package installed in their env). If they also have `langchain` package v0.0.310 installed explicitly or for some other reason, it's fine, these are not clashing.

- However, if users do not install `langfuse` with the `langchain` extra and do not have `langchain-core` in the env for some other reason, the upgrade will not work for them. I would say that is expected and correct though (if you don't specify you want to use langfuse with langchain, then langfuse<>langchain might not be compatible).

Reply:
bump on this? it's a minor inconvenience, but I'd love to see it merged